s2brdf/kernels.py

```diff
@@ -14,7 +14,7 @@ def rad2deg(x):
 def relative_azimuth(saa, vaa):
     """Calculate the relative azimuth angle between a SAA and VAA.
        In Radians."""
-    return abs(vaa - pi - saa)
+    return abs(vaa - saa)
```

Comment:
Relative angle should be normalised to 360%.

We could do:

```
return (vaa-saa) % 2*pi
```

Reply:
That form of `relative_azimuth` still produces good BRDF correction results, but but since we are already working with radians I believe we do not need to make adjustments for negative values or values over 2 * pi, right? 

Is the principal plane the case where SAA and VAA are equal? In Roy et al 2016 I see this in the caption for Fig 2: "..in the principal plane (0° relative azimuth)". If I set the SAA and VAA equal to one another for the principal plane examples in test_kernels.py and in the method comparison notebook and use `abs(vaa - saa)` for the relative azimuth equation, everything works as expected and the plotted lines all line up pretty well.

Pardon my ignorance on some of the remote sensing basics...

Reply:
I clearly wasn't thinking straight yesterday: of course you don't need to normalise the RAA :roll_eyes:  However you do need to use the absolute difference `abs(vaa-saa)`. 

The principle plane is usually the azimuth transect 0-180° relative to a reference. For example, when characterising the BRDF of materials (I worked on sea-ice in my PhD thesis) you often refer to the SPP (solar principle plane) because the illumination is fixed and you plot the response over all the viewing angles. In this case, the Solar Principle Plane is the azimuth towards and away from the sun. As a side note, for most materials, you can get away with mirroring each side of the SPP making it easier to characterise.

![image](https://user-images.githubusercontent.com/32132761/133746508-43972655-d198-4195-abcc-45dfb567c22d.png)

The cross-plane is the 90-270 azimuth line in regards to a reference.

Here is an example of a BRDF (to use vague terminology; for exact terminology see [this paper](https://www.sciencedirect.com/science/article/pii/S0034425706001167)) plot of  ice clouds from [Gatebe and King](https://www.sciencedirect.com/science/article/pii/S0034425716301237) to illustrate the concept. That might help.

![image](https://user-images.githubusercontent.com/32132761/133746957-8a15ad85-9410-424f-b78d-e13bc38130db.png)

In our case, I'm assuming Principle Plane will refer to the azimuth where Illumination = Viewing.